<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
         <x>0</x>
         <y>0</y>
         <width>761</width>
         <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>440</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>please hear</string>
     </property>
    </widget>
       <widget class="QTextBrowser" name="textBrowser">
           <property name="geometry">
               <rect>
                   <x>250</x>
                   <y>30</y>
                   <width>481</width>
                   <height>391</height>
               </rect>
           </property>
           <property name="accessibleDescription">
               <string notr="true"/>
           </property>
           <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
               <string notr="true">
                   background-color: rgba(255, 255, 255,0);
               </string>
           </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
               <rect>
                   <x>630</x>
                   <y>430</y>
                   <width>111</width>
                   <height>81</height>
               </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                   <widget class="QPushButton" name="pushButton">
                       <property name="text">
                           <string>发送文件</string>
                       </property>
                   </widget>
               </item>
               <item>
                   <widget class="QPushButton" name="unknow">
                       <property name="text">
                           <string>發送(alt+s)</string>
                       </property>
                   </widget>
               </item>
           </layout>
       </widget>
       <widget class="QLabel" name="label_3">
           <property name="geometry">
               <rect>
                   <x>30</x>
                   <y>0</y>
                   <width>41</width>
                   <height>28</height>
               </rect>
           </property>
           <property name="font">
               <font>
                   <pointsize>14</pointsize>
               </font>
           </property>
           <property name="text">
               <string>IM</string>
           </property>
       </widget>
       <widget class="QLabel" name="myName">
           <property name="geometry">
               <rect>
                   <x>60</x>
                   <y>0</y>
                   <width>181</width>
                   <height>28</height>
               </rect>
           </property>
           <property name="font">
               <font>
                   <pointsize>17</pointsize>
               </font>
           </property>
           <property name="text">
               <string>I m</string>
           </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
               <rect>
                   <x>240</x>
                   <y>320</y>
                   <width>120</width>
                   <height>191</height>
               </rect>
           </property>
           <property name="styleSheet">
               <string notr="true">background-color: rgba(143, 207, 255,100);</string>
           </property>
           <property name="inputMethodHints">
               <set>Qt::ImhUrlCharactersOnly</set>
           </property>
           <property name="title">
               <string/>
           </property>
           <widget class="QPushButton" name="dir_send">
               <property name="geometry">
                   <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>121</width>
                       <height>31</height>
                   </rect>
               </property>
               <property name="text">
                   <string>文件夹</string>
               </property>
           </widget>
           <widget class="QPushButton" name="pushButton_2">
               <property name="geometry">
                   <rect>
                       <x>0</x>
                       <y>120</y>
                       <width>61</width>
                       <height>71</height>
                   </rect>
               </property>
               <property name="text">
                   <string>___</string>
               </property>
           </widget>
           <widget class="QPushButton" name="handWrite">
               <property name="geometry">
                   <rect>
                       <x>0</x>
                       <y>30</y>
                       <width>121</width>
                       <height>28</height>
                   </rect>
               </property>
               <property name="text">
                   <string>手写</string>
               </property>
           </widget>
       </widget>
       <widget class="QPushButton" name="showGroup">
           <property name="geometry">
               <rect>
                   <x>240</x>
                   <y>440</y>
                   <width>61</width>
                   <height>71</height>
               </rect>
           </property>
           <property name="text">
               <string>___</string>
           </property>
       </widget>
       <widget class="QFrame" name="frame_2">
           <property name="geometry">
               <rect>
                   <x>250</x>
                   <y>30</y>
                   <width>481</width>
                   <height>391</height>
               </rect>
           </property>
           <property name="styleSheet">
               <string notr="true">background-image: url(:/images/images/abc.jpg);</string>
           </property>
           <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
               <enum>QFrame::Raised</enum>
           </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
               <rect>
                   <x>130</x>
                   <y>460</y>
                   <width>111</width>
                   <height>51</height>
               </rect>
           </property>
           <property name="text">
               <string>好友</string>
           </property>
       </widget>
       <widget class="QScrollArea" name="frindsArea">
           <property name="geometry">
               <rect>
                   <x>10</x>
                   <y>40</y>
                   <width>221</width>
                   <height>381</height>
               </rect>
           </property>
           <property name="widgetResizable">
               <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                   <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>219</width>
                       <height>379</height>
                   </rect>
               </property>
               <widget class="QScrollBar" name="verticalScrollBar">
                   <property name="geometry">
                       <rect>
                           <x>200</x>
                           <y>10</y>
                           <width>16</width>
                           <height>361</height>
                       </rect>
                   </property>
                   <property name="orientation">
                       <enum>Qt::Vertical</enum>
                   </property>
               </widget>
           </widget>
       </widget>
       <widget class="QFrame" name="frame_3">
           <property name="geometry">
               <rect>
                   <x>20</x>
                   <y>430</y>
                   <width>91</width>
                   <height>81</height>
               </rect>
           </property>
           <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
               <enum>QFrame::Raised</enum>
           </property>
           <widget class="QPushButton" name="pushButton_4">
               <property name="geometry">
                   <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>93</width>
                       <height>91</height>
                   </rect>
               </property>
               <property name="styleSheet">
                   <string notr="true">background-color: transparent;</string>
               </property>
               <property name="text">
                   <string/>
               </property>
           </widget>
       </widget>
       <zorder>frame_2</zorder>
       <zorder>textBrowser</zorder>
       <zorder>showGroup</zorder>
       <zorder>layoutWidget</zorder>
       <zorder>lineEdit</zorder>
       <zorder>label_3</zorder>
       <zorder>myName</zorder>
       <zorder>groupBox</zorder>
       <zorder>pushButton_3</zorder>
       <zorder>frindsArea</zorder>
       <zorder>frame_3</zorder>
   </widget>
      <widget class="QScrollArea" name="filePersents">
          <property name="geometry">
              <rect>
                  <x>750</x>
                  <y>10</y>
                  <width>221</width>
                  <height>521</height>
              </rect>
          </property>
          <property name="widgetResizable">
              <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
                  <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>219</width>
                      <height>519</height>
                  </rect>
              </property>
              <widget class="QScrollBar" name="verticalScrollBar_2">
                  <property name="geometry">
                      <rect>
                          <x>180</x>
                          <y>10</y>
                          <width>16</width>
                          <height>511</height>
                      </rect>
                  </property>
                  <property name="orientation">
                      <enum>Qt::Vertical</enum>
                  </property>
              </widget>
          </widget>
      </widget>
  </widget>
     <widget class="QMenuBar" name="menubar">
         <property name="geometry">
             <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>982</width>
                 <height>26</height>
             </rect>
         </property>
         <widget class="QMenu" name="menuyu">
             <property name="title">
                 <string>設置</string>
             </property>
             <addaction name="actionhow"/>
             <addaction name="actionself_info"/>
             <addaction name="actiontext_font"/>
         </widget>
         <addaction name="menuyu"/>
     </widget>
     <widget class="QStatusBar" name="statusbar"/>
     <action name="actionhow">
         <property name="text">
             <string>name</string>
         </property>
     </action>
     <action name="actionself_info">
         <property name="text">
             <string>self_info</string>
         </property>
     </action>
     <action name="actionfiel_save_path">
         <property name="text">
             <string>fiel_save_path</string>
         </property>
     </action>
     <action name="actiontext_font">
         <property name="text">
             <string>text_font</string>
         </property>
     </action>
 </widget>
    <resources>
        <include location="res.qrc"/>
    </resources>
    <connections>
        <connection>
            <sender>showGroup</sender>
            <signal>clicked()</signal>
            <receiver>groupBox</receiver>
            <slot>show()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>270</x>
                    <y>501</y>
                </hint>
                <hint type="destinationlabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>dir_send</sender>
            <signal>clicked()</signal>
            <receiver>groupBox</receiver>
            <slot>hide()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>300</x>
                    <y>361</y>
                </hint>
                <hint type="destinationlabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>groupBox</sender>
            <signal>clicked()</signal>
            <receiver>groupBox</receiver>
            <slot>hide()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
                <hint type="destinationlabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>pushButton_2</sender>
            <signal>clicked()</signal>
            <receiver>groupBox</receiver>
            <slot>hide()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>270</x>
                    <y>501</y>
                </hint>
                <hint type="destinationlabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>handWrite</sender>
            <signal>clicked()</signal>
            <receiver>groupBox</receiver>
            <slot>hide()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>300</x>
                    <y>389</y>
                </hint>
                <hint type="destinationlabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>textBrowser</sender>
            <signal>selectionChanged()</signal>
            <receiver>groupBox</receiver>
            <slot>hide()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>500</x>
                    <y>266</y>
                </hint>
                <hint type="destinationlabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>lineEdit</sender>
            <signal>selectionChanged()</signal>
            <receiver>groupBox</receiver>
            <slot>hide()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>460</x>
                    <y>501</y>
                </hint>
                <hint type="destinationlabel">
                    <x>299</x>
                    <y>441</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
